module1.js

```diff
@@ -17,18 +18,21 @@
 */
 
 function question1(name1) {
-  if (name1) {
-    // TODO:
-    return;
+  if (name1 === 'Code Your Future') {
+    return true;
+  } else {
+    return false;
   }
 }
 
```

Comment:
Good. You could also use the rather shorter
```
return ( name1 === 'Code Your Future');
```
which will give either true or false